src/main/scala/org/constellation/infrastructure/endpoints/SnapshotEndpoints.scala

```diff
@@ -193,15 +200,17 @@ class SnapshotEndpoints[F[_]](implicit F: Concurrent[F]) extends Http4sDsl[F] {
             case Invalid(IncorrectSenderId(_))                                                  => Response[F](status = Unauthorized).pure[F]
             case Invalid(e)                                                                     => logger.error(e)(e.getMessage) >> InternalServerError()
             case Valid(_) =>
+              val processProposalAsync = F.start(
+                C.shift >>
```

Comment:
same q as before

Reply:
It's the same place as before. Everything still happens on the `unbounded-pool-1`.